lib/excoveralls/circle.ex

```diff
@@ -7,14 +7,14 @@ defmodule ExCoveralls.Circle do
   def execute(stats, options) do
     json = generate_json(stats, Enum.into(options, %{}))
     if options[:verbose] do
-      IO.puts JSX.prettify!(json)
```

Comment:
Only thing that we drop then is pretty printing which is not that bad. We can get it back after Jason implements it https://github.com/michalmuskala/jason/issues/15